cinn/ir/ir_schedule.cc

```diff
@@ -1201,6 +1201,58 @@ Expr IRSchedule::CacheWrite(const Expr& block, int write_buffer_index, const std
   return *find_cache_block.begin();
 }
 
+struct InsertExpr : public ir::IRMutator<> {
+ public:
+  static void Insert(Expr* expr, const Expr& ir_node, const Expr& insert_node, bool after_node) {
+    InsertExpr mutator(ir_node, insert_node, after_node);
+    mutator(expr);
+  }
+
+  void operator()(Expr* expr) { IRMutator::Visit(expr, expr); }
+
+ private:
+  explicit InsertExpr(const Expr& ir_node, const Expr& insert_node, bool after_node)
+      : ir_node_(ir_node), insert_node_(insert_node), after_node_(after_node) {}
+
+  void Visit(const ir::Block* expr, Expr* op) override {
+    for (int i = 0; i < expr->stmts.size(); i++) {
+      if (expr->stmts[i] == ir_node_) {
+        if (after_node_)
+          op->As<ir::Block>()->stmts.insert(op->As<ir::Block>()->stmts.begin() + i + 1, insert_node_);
```

Comment:
Add brace for coding style.

Reference: https://google.github.io/styleguide/cppguide.html#Conditionals

Reply:
Done, thanks